lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb

```diff
@@ -117,6 +112,45 @@ def perform
 
   private
 
+  # Look for the location of EnergyPlus
+  def find_energyplus
+    if ENV['ENERGYPLUSDIR']
+      return ENV['ENERGYPLUSDIR']
+    elsif ENV['RUBYLIB'] =~ /OpenStudio/
```

Comment:
@macumber, I am not happy about this method and it really isn't finished yet. Can we chat about this? My thinking is that if we are shipping energyplus with openstudio, then let's just use that one. If so, then is there a method in openstudio to get the energyplus fqp?